CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.1.0] - 2021-07-16
+
 * Add `BakeNoteIFrames` direction (minor)
 * Selector optimization in precalculus `Strategy` & elsewhere (patch)
 * Tweaks inline list seperators to only target labeled items (major?)
```

Comment:
ooh do you mind changing this to minor? i double-checked and it doesn't look like it'll impact any published books

Reply:
oops - saw ur comment right after i hit merge lol
I assumed it was minor so only did the 9.1 instead of 10.0